src/Logger.php

```diff
@@ -29,6 +29,15 @@ class Logger implements LoggerInterface
 {
     use LoggerTrait;
 
+    public const EMERGENCY = LogLevel::EMERGENCY;
```

Comment:
Shouldn't this be prefixed/suffixed by `LEVEL`? `LogLevel::EMERGENCY` is obviois, `Logger::EMERGENCY` is not.

Reply:
Suffixed for better completion